attributes/default.rb

```diff
@@ -20,6 +20,8 @@
 # not required for later versions
 default['cassandra']['setup_jamm'] = false
 
+default['cassandra']['priam']['version'] = ''
```

Comment:
`nil` value is more preferable than an empty string. i think it would be better to set `default` version and also add `priam.jar` file download and setup in the cookbook.